beacon-chain/sync/validate_aggregate_proof.go

```diff
@@ -45,9 +45,14 @@ func (s *Service) validateAggregateAndProof(ctx context.Context, pid peer.ID, ms
 	if !ok {
 		return pubsub.ValidationReject
 	}
-	if m.Message == nil || m.Message.Aggregate == nil || m.Message.Aggregate.Data == nil {
+	if m.Message == nil {
 		return pubsub.ValidationReject
 	}
+	if err := helpers.ValidateNilAttestation(m.Message.Aggregate); err != nil {
+		log.WithError(err).Debug("Invalid attestation")
```

Comment:
Should we remove this debug message? It might flood logs and I believe we log or handle pubsub rejections already.

Reply:
Good idea